deep_qa/data/instances/tuple_inference_instance.py

```diff
@@ -22,11 +22,11 @@ def __init__(self, subject: str, verb: str, objects: List[str],
 
     def display_string(self):
         # NOTE: not currently displaying location, time, and source info.
-        return ("TextTuple( [\n\tsubj: {0} \n\tverb: {1} \n\tobjects: {2} " +
-                "\n\tcontext: {3}\n] )").format(self.subject,
-                                                self.verb,
-                                                ", ".join(self.objects),
-                                                self.context)
+        return ("\tTextTuple( [\n\t\tsubj: {0} \n\t\tverb: {1} \n\t\tobjects: {2} " +
```

Comment:
I found this gave a more compact list -- maybe harder to read in detail -- so use at will:
```
return ("(S: {0}, V: {1}, O: {2}, C: {3})").format(self.subject, self.verb, ", ".join(self.objects), 
self.context[:100])
```